From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] Fixes for Visual Studio 18 2026

stdfloat misbehaves in VS2026 for some reason I don't remember. We can build without it just fine.

---

diff --git a/third_party/fast_float/src/include/fast_float/float_common.h b/third_party/fast_float/src/include/fast_float/float_common.h
index 1265d9e..c826b95 100644
--- a/third_party/fast_float/src/include/fast_float/float_common.h
+++ b/third_party/fast_float/src/include/fast_float/float_common.h
@@ -7,11 +7,6 @@
 #include <limits>
 #include <type_traits>
 #include <system_error>
-#ifdef __has_include
-#if __has_include(<stdfloat>) && (__cplusplus > 202002L || (defined(_MSVC_LANG) && (_MSVC_LANG > 202002L)))
-#include <stdfloat>
-#endif
-#endif
 #include "constexpr_feature_detect.h"
 
 #define FASTFLOAT_VERSION_MAJOR 8
